packages/analytics-js-integrations/__tests__/utils/commonUtils.test.js

```diff
@@ -345,3 +345,14 @@ describe('pick', () => {
     expect(result).toEqual({ a: 1, c: 3, A: 4, C: 6 });
   });
 });
+
+describe('isBlank', () => {
```

Comment:
The test scenarios do not properly cover the implementation.

Add the missing test cases:
- `     ` (all whitespace)
- `asdf` (a non-empty string)
- Maybe, also, check for more data types like numbers, objects, etc.

Add any other scenarios that make sense.

Reply:
Updated.